specification/edgeorder/EdgeOrder.Management/back-compatible.tsp

```diff
@@ -0,0 +1,95 @@
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+using Microsoft.EdgeOrder;
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(Configuration.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(ChildConfiguration.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(ProductFamily.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(ProductLine.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(Product.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(ProductFamiliesMetadataDetails.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(AddressUpdateParameter.properties);
+
+#suppress "deprecated" "FIXME: Update justification, follow aka.ms/tsp/conversion-fix for details"
+@@flattenProperty(OrderItemUpdateParameter.properties);
```

Comment:
Are the real justifications for now backcompatibility?

But is the idea to fix these in the future? Since they are a linter warning??

How is this actually going to happen? /cc @MattGertz

Reply:
We will add document to the link aka.ms/tsp/conversion-fix